tests/bazel_tests.bzl

```diff
@@ -142,8 +142,7 @@ def _bazel_test_script_impl(ctx):
     register = ""
     if any([ext.label.workspace_root.endswith("/go_sdk") for ext in ctx.attr.externals]):
         register += "register_toolchains(\n{}\n)\n".format(
-            "\n".join(['    "@go_sdk//:{}",'.format(name)
-                       for name in generate_toolchain_names()]))
+            "\n".join(['"@go_sdk//:{}",'.format(name) for name in generate_toolchain_names()]))
```

Comment:
Why remove the indentation in the generated call to `register_toolchains`? It's not strictly necessary, but it makes it more readable.